src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/RefreshVendorRegistrationCaseStatus.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntities;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities
+{
+    public class RefreshVendorRegistrationCaseStatus
+    {
+        private readonly IVendorRegistrationFormService _vendorRegistrationFormService;
+        private readonly IVrfCaseRefreshConfiguration _configuration;
+        private readonly IDateTimeProvider _dateTimeProvider;
+
+        public RefreshVendorRegistrationCaseStatus(IVendorRegistrationFormService vendorRegistrationFormService,
+            IVrfCaseRefreshConfiguration configuration,
+            IDateTimeProvider dateTimeProvider)
+        {
+            _vendorRegistrationFormService = vendorRegistrationFormService;
+            _configuration = configuration;
+            _dateTimeProvider = dateTimeProvider;
+        }
+
+        // Every hour: "* 0 * * * *"
+        [FunctionName("RefreshVendorRegistrationCaseStatus")]
+        public async Task Run([TimerTrigger("%RefreshVendorRegistrationCaseStatusTriggerTime%")] TimerInfo myTimer, ILogger log)
```

Comment:
Inject the logger in the constructor.

Reply:
this is a standard practice as per microsoft's documentation. logger here gets injected automagically and it's consistent with how this function was done before  :)

Reply:
That's before .net core supported DI.